docs/contentSharing.md

```diff
@@ -76,3 +76,8 @@ Just as anyone with access to a question in your course can access any file in `
 3. Add `"sharePublicly": true` to the `info.json` file of questions you would like to share publicly.
 4. Optionally, add `"shareSourcePublicly": true` to the `info.json` file if you would like people to also be able to view and copy the source code of your question.
 5. Anyone with a PrairieLearn account may preview your question, and any PrairieLearn course may now use it by referencing it in their assessments.
+
+## Steps to share an assessment
+
+1. Ensure that all questions on the assessment have `"sharePublicly": true`. This is to enable others to copy your assessment into their course and maintain access to all of its questions (we may support copying questions and assessments together in the future, but for now only the assessment will be copied).
```

Comment:
We prefer not to talk about what may be possible in the future in the docs. It's better to talk directly in terms of what is or is not supported at this moment.

```suggestion
1. Ensure that all questions on the assessment have `"sharePublicly": true`. This is to enable others to copy your assessment into their course and maintain access to all of its questions. Currently, copying the questions as well as the assessment is not supported; the copied assessment will always include the shared questions by reference.
```

Feel free to workshop the text as you see fit.

Reply:
done in c20ad5e7c2